packages/mobile/src/send/TransferConfirmationCard.tsx

```diff
@@ -1,86 +1,30 @@
 import { Avatar } from '@celo/react-components/components/Avatar'
```

Comment:
This file was previously responsible for the 'card' content in both the send flow and the tx drilldown from wallet home. Supporting both made it more complex than it had to be and made it painful to diverge those designs. Now this component is for drilldown and `TransferReviewCard` is for the send flow